src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -62,6 +63,14 @@ public static String getEditPersonDescriptorDetails(EditPersonDescriptor descrip
                 tags.forEach(s -> sb.append(PREFIX_TAG).append(s.tagName).append(" "));
             }
         }
+        if (descriptor.getGradedTests().isPresent()) {
+            Set<GradedTest> gradedTests = descriptor.getGradedTests().get();
```

Comment:
Is this preferred over having the descriptor to return the Set<GradedTest> instead for SLAP?